DEPENDENCIES

```diff
@@ -305,6 +305,9 @@ Prerequisites for building the OpenDataPlane (ODP) API
    e.g. with some packet length, packet segment length and pool size
    combinations that would otherwise conform to reported capabilities.
 
+   Note that, currently, AF_XDP socket packet I/O cannot be instantiated if
+   DPDK zero-copy is configured.
```

Comment:
Perhaps `enabled` would be a better word, since it looks like XDP can still be used if DPDK zero-copy pktio is configured and disabled.

Reply:
Yes, that's more accurate, fixed in v2.